spec/runners/unified.rb

```diff
@@ -69,11 +69,11 @@ def define_unified_spec_tests(base_path, paths, expect_failure: false)
                   test.assert_events
                 # HACK: other errors are possible and likely will need to
                 # be added here later as the tests evolve.
-                rescue Mongo::Error::OperationFailure, Unified::Error::UnsupportedOperation
+                rescue ArgumentError, Mongo::Error::OperationFailure, Unified::Error::UnsupportedOperation, Unified::Error::EntityMissing
                 rescue => e
-                  fail "Expected to raise Mongo::Error::OperationFailure or Unified::Error::UnsupportedOperation, got #{e}"
+                  fail "Expected to raise ArgumentError, Mongo::Error::OperationFailure, Unified::Error::UnsupportedOperation or Unified::Error::EntityMissing, got #{e}"
                 else
-                  fail "Expected to raise Mongo::Error::OperationFailure or Unified::Error::UnsupportedOperation, but no error was raised"
+                  fail "Expected to raise error, but none was raised"
```

Comment:
Why change to this message?

Reply:
It was just getting really long and I didn't feel like it was necessary to list all the errors if none were raised.